### PR TITLE
feat(dedicated.network-security): tracking plan

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-498
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

add tracking plan for Edge firewall, GAME firewall and Network Security Dashboard

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js [hunk 1]
```diff
@@ -5,16 +5,27 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
...
   const alert = 'ip_game_firewall_alert';
 
   self.datas = $scope.currentActionData;
   self.loading = false;
 
+  atInternet.trackClick({
+    name: $scope.data?.tracking,
+    type: 'action',
+  });
+
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js:20]: I think you should test if $scope.data?.tracking is not undefined before calling atInternet.trackClick with it.
Unless we are sure data will allways be defined and has tracking also defined.

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js [hunk 2]
```diff
@@ -5,16 +5,27 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
...
   $scope.removeGameFirewallRule = function removeGameFirewallRule() {
     self.loading = true;
 
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}-confirm`,
+      type: 'action',
+    });
+
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js:28]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js [hunk 3]
```diff
@@ -43,4 +54,12 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
         $scope.resetAction();
       });
   };
+
+  $scope.cancelDelete = function cancelDelete() {
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}-cancel`,
+      type: 'action',
+    });
+    $scope.resetAction();
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js:63]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ export default /* @ngInject */ (
   });
   $scope.removeRule = function removeRule() {
     atInternet.trackClick({
-      name: `${$scope.data?.tracking}::confirm`,
+      name: `${$scope.data?.tracking}-confirm`,
       type: 'action',
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js:18]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js [hunk 5]
```diff
@@ -40,7 +40,7 @@ export default /* @ngInject */ (
   };
   $scope.cancelAction = function cancelAction() {
     atInternet.trackClick({
-      name: `${$scope.data?.tracking}::cancel`,
+      name: `${$scope.data?.tracking}-cancel`,
       type: 'action',
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js:44]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/toggle/ip-ip-firewall-toggle.controller.js [hunk 6]
```diff
@@ -13,10 +13,12 @@ export default /* @ngInject */ (
...
     $scope.data = data || $scope.currentActionData;
-    atInternet.trackPage({
-      name: $scope.data?.tracking,
-    });
+    const tracking = $scope.data?.tracking;
     if ($scope.data.ip.firewall === 'ACTIVATED') {
+      atInternet.trackClick({
+        name: `${tracking}-off`,
+        type: 'action',
+      });
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/toggle/ip-ip-firewall-toggle.controller.js:21]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/toggle/ip-ip-firewall-toggle.controller.js [hunk 7]
```diff
@@ -27,6 +29,10 @@ export default /* @ngInject */ (
         },
       );
     } else if ($scope.data.ip.firewall === 'DEACTIVATED') {
+      atInternet.trackClick({
+        name: `${tracking}-on`,
+        type: 'action',
+      });
```
Comment (JacquesLarique) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/toggle/ip-ip-firewall-toggle.controller.js:35]: same as https://github.com/ovh/manager/pull/10520#discussion_r1425095934

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js [hunk 8]
```diff
@@ -5,16 +5,27 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
...
+  atInternet.trackClick({
+    name: $scope.data?.tracking,
+    type: 'action',
+  });
+
   $scope.removeGameFirewallRule = function removeGameFirewallRule() {
     self.loading = true;
 
+    atInternet.trackClick({
+      name: `${$scope.data?.tracking}-confirm`,
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js:25]: in that way, you're sending `null-confirm` as a potential value 
```suggestion
      name: $scope.data?.tracking} ? `${$scope.data?.tracking}-confirm` : null,
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js [hunk 9]
```diff
@@ -9,12 +9,12 @@ export default /* @ngInject */ (
 ) => {
   $scope.data = $scope.currentActionData;
   atInternet.trackClick({
-    name: $scope.data?.tracking,
+    name: $scope.data?.tracking ? $scope.data?.tracking : null,
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js:12]:
```suggestion
    name: $scope.data?.tracking || null,
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js [hunk 10]
```diff
@@ -9,12 +9,12 @@ export default /* @ngInject */ (
...
     type: 'action',
   });
   $scope.removeRule = function removeRule() {
     atInternet.trackClick({
-      name: `${$scope.data?.tracking}::confirm`,
+      name: $scope.data?.tracking ? `${$scope.data?.tracking}-confirm` : null,
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js:17]:
```suggestion
      name: $scope.data?.tracking ? `${$scope.data.tracking}-confirm` : null,
```

File: packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js [hunk 11]
```diff
@@ -40,7 +40,7 @@ export default /* @ngInject */ (
   };
   $scope.cancelAction = function cancelAction() {
     atInternet.trackClick({
-      name: `${$scope.data?.tracking}::cancel`,
+      name: $scope.data?.tracking ? `${$scope.data?.tracking}-cancel` : null,
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/ip/components/list/firewall/rule/delete/ip-ip-firewall-rule-delete.controller.js:43]:
```suggestion
      name: $scope.data?.tracking ? `${$scope.data.tracking}-cancel` : null,
```

**Review by JacquesLarique — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**